application-admintools-ui/src/main/resources/AdminTools/EmptyPages.xml

```diff
@@ -39,13 +39,14 @@
   <content>{{velocity}}
 {{html clean='false' wiki='true'}}
   #set ($sourceParameters = $escapetool.url({
+    'className': 'AdminTools.SpammedPages',
```

Comment:
It's a little bit strange to use the `SpammedPages` object in here. Maybe you could have a separate page for this object?